### PR TITLE
Fixing issues on Date Validation.

### PR DESCRIPTION
**Pull request by slubwama**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml [hunk 1]
```diff
@@ -44,10 +42,15 @@
             submitValStage=submitValStageART;
 
             submitValStage.push(dateValidator("90267", "encounterDate", "", message4, "", "greater_than", true));
-            submitValStage.push(dateValidator("90297", "encounterDate", "", message1, "", "less_than", true));
-            submitValStage.push(dateValidator("90299", "90297", "encounterDate", message2, message1, "less_than", false));
             submitValStage.push(dateValidator("99161", "90299", "encounterDate", message3, message1, "less_than", false));
-
+            if (submitValStage.indexOf(false) != -1) {
+                submitVal = false;
+                jq().toastmessage('showErrorToast', "Form Validation violated. Check for errors");
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml]: @slubwama 

1. "Violated" I think It should be validated

2. Why return here when you still have other checks at the bottom?

**Comment by ssmusoke** [comment]
@slubwama In case of Transfer Ins - the Eligible for ART, and Eligible and ready dates can be after the HIV Enrolled date, since this is the date that the client enrolled in the current facility

**Comment by ssmusoke** [comment]
@slubwama Change message popup to say - "There are errors during the valdation of the data on the form. Please correct them and submit again" instead of "Form Validation Violated" which is not user friendly.

**Comment by slubwama** [comment]
2. That message only appears if art start date and art transfer date are missing.

**Pull request merged**